### PR TITLE
Disable mempool config

### LINKED ISSUE #2090: Backend config to disable the mempool
Since we run Lightning and other(?) backends without using the mempool, the main mempool loop should be disable-able.

Create a new config to disable the mempool loop that is disabled by default.

Check what else could be turned off by the same config.

### PR DESCRIPTION
**Pull request by softsimon**
fixes #2090

New config option that disables all mempool, block indexing and mempool stats activity including the routes. Intended for Lightning backend, default is enabled = true.

```
  "MEMPOOL": {
    "ENABLED": false,
```

### CODE REVIEW

**Comment by nymkappa** [comment]
Tested ACK @ 9c0542351f67a95a3da4ec3d0a9e0786e73486c9

**Review by nymkappa — CHANGES_REQUESTED** [review]
You did not update docker configuration files but it works fine Can be merged once files are updated.

Tested ACK @ 9c0542351f67a95a3da4ec3d0a9e0786e73486c9

**Comment by softsimon** [comment]
Added docker config. Hope I did it right, just played by the ear.

**Comment by nymkappa** [comment]
You're missing `mempool/docker/README.md` but I think otherwise you're good. @knorrium may double check as well.

**Review by nymkappa — CHANGES_REQUESTED** [review]
You're missing `mempool/docker/README.md` but I think otherwise you're good. @knorrium may double check as well.

**Review by nymkappa — APPROVED** [review]

File: backend/src/__fixtures__/mempool-config.template.json [hunk 1]
```diff
@@ -1,7 +1,9 @@
 {
   "MEMPOOL": {
+    "ENABLED": true,
     "NETWORK": "__MEMPOOL_NETWORK__",
     "BACKEND": "__MEMPOOL_BACKEND__",
+    "ENABLED": true,
```
Comment (knorrium) [inline backend/src/__fixtures__/mempool-config.template.json:6]:
```suggestion

```

Comment (knorrium) [reply backend/src/__fixtures__/mempool-config.template.json:6]: This is duplicated above

**Review by knorrium — CHANGES_REQUESTED** [review]

**Review by wiz — APPROVED** [review]
Tested ACK @ 5957b71774e16d027919a531a1854e31bd1a7515

**Pull request merged**